enforcer/src/main/java/org/wso2/choreo/connect/enforcer/config/ConfigHolder.java

```diff
@@ -191,9 +191,14 @@ private void populateJWTIssuerConfiguration(Security security) {
             // Load jwt transformers map.
             config.setJwtTransformerMap(JWTUtil.loadJWTTransformers());
             String certificateAlias = jwtIssuer.getCertificateAlias();
-            if (certificateAlias.isBlank() && APIConstants.KeyManager.DEFAULT_KEY_MANAGER.equals(jwtIssuer.getName())) {
-                certificateAlias = APIConstants.GATEWAY_PUBLIC_CERTIFICATE_ALIAS;
+            if (certificateAlias.isBlank()) {
```

Comment:
if certificateAlias is null, then this will throw NPE

Reply:
Could this be removed from parent configuration? +1 to check null

Reply:
certificateAlias couldn't be null. It always not null as only can be empty string